### ISSUE TITLE
why is the data of val_loss very big?

### ISSUE THREAD

**Issue by Simon9901** [NICE-FUTURE/predict-gender-and-age-from-camera#4]
Hello,when I was training the network,I found that the loss is very small,but the data of val_loss is pretty big, so I cannot draw the correct the picture of loss-val_loss.The questional process is as follows.
![image](https://user-images.githubusercontent.com/103345899/166700758-3434b981-e8b8-485c-aa3c-a6b89933a8a6.png)

At the same time,I also want to learn about how the pretrained weight should be used.

Thank you very much!

**Comment by NICE-FUTURE** [comment]
Hello~ I checked my training log at [here](https://github.com/NICE-FUTURE/predict-gender-and-age-from-camera/blob/master/middle/logs/test-2022-04-07%2011.35.16.210418.log), and I did not find the problem you mentioned above.

1. Maybe you can print out the "age_loss" and "gender_loss" respectively. If the "age_loss" is unnormal, you can check the age label which produced at [this line in dataset.py](https://github.com/NICE-FUTURE/predict-gender-and-age-from-camera/blob/8b469cb5979b476d80c83a80a6b1440ba96f534c/datasets.py#L19). If the "gender_loss" is unnormal, you can check the gender label which produced at [this line in dataset.py](https://github.com/NICE-FUTURE/predict-gender-and-age-from-camera/blob/8b469cb5979b476d80c83a80a6b1440ba96f534c/datasets.py#L20). _(In order to normalize the value of age, I divided the age by 100 directly, there might be a better alternative.)_

2. The pretrained weights can be used to reproduce the results I posted in [the end of README.md](https://github.com/NICE-FUTURE/predict-gender-and-age-from-camera#%E6%A8%A1%E5%9E%8B%E6%95%88%E6%9E%9C). But I don't think the weights I provided are good initial weights in the training stage, the default ImageNet pretrained weights are better choice.

Best regards.
